src/interface/src/app/services/treatment-to-pdf.service.ts

```diff
@@ -0,0 +1,179 @@
+import { Injectable } from '@angular/core';
+import jsPDF from 'jspdf';
+import autoTable from 'jspdf-autotable';
+import {
+  PRESCRIPTIONS,
+  PrescriptionSequenceAction,
+  nameForTypeAndAction,
+} from '../treatments/prescriptions';
+import { Map as MapLibreMap } from 'maplibre-gl';
```

Comment:
One catch here- adding this dependency here, and using providedIn: root might cause the compiler to bundle maplibre as part of the main bundle, which we dont want.

An alternative, remove providedIn root and provide this service on the treatments module, and move to treatments.

You can use https://www.npmjs.com/package/webpack-bundle-analyzer or any other bundle analyser to check this (or diff main with this branch on the bundle generation output when building)